Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian
+
+RUN apt update &&\
+        apt -y install git curl gnupg2 &&\
+        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - &&\
+        echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list &&\
+        apt update && apt -y install yarn &&\
+        git -C /root clone https://github.com/jobertabma/transformations &&\
+        cd /root/transformations/ &&\
+        yarn install &&\
+        yarn build
```

Comment:
`yarn start` doesn't serve the static build. If you want to serve it, I'd suggest that we run `nginx` (also used in production) and point nginx to serve the static build instead, which is located in /root/transformations/build. How does that sound? Can you also add a small README.md for others so they know how to run it in Docker? I can clean up the README after this is merged. Thanks!

Reply:
Of course! Let me push a commit to my master branch and tell me what you think